_data/simple-icons.json

```
@@ -11289,6 +11289,15 @@
             "source": "https://brand.unity.com/",
             "guidelines": "https://brand.unity.com/"
         },
+        {
+            "title": "Unlicense",
+            "hex": "808080",
+            "source": "https://commons.wikimedia.org/wiki/File:PD-icon.svg",
+            "guidelines": "https://unlicense.org/",
+            "license": {
+                "type": "Unlicense"
+            }
```

The Unlicense website doesn't contain any guidelines regarding the usage of the _icon_, only information about the license itself, unless I missed or misunderstood something.

Also, I can't find any indication that the _icon_ is licensed under the Unlicense.

```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:PD-icon.svg"
```

---

The icon is likely Public Domain. 
I misunderstood what the guidelines field was for